graph.py

```diff
@@ -60,3 +60,42 @@ def adjacent(self, n1, n2):
             return True
         else:
             return False
+
+    def breadth_first_traversal(self, start):
+        visited = set()
+        queue = [start]
+        return_value = []
+        while queue:
+            node = queue.pop(0)
```

Comment:
Because popping a Python list at index 0 is an O(n) operation, it would be enormously better to use an actual queue.  You can use the one you built before, or you can use the `deque` object that comes from the `collections` standard library module.  That will allow this operation to be O(1)